scripts/run_ci_unit_coverage.sh

```diff
@@ -1,12 +0,0 @@
- #!/bin/bash
-touch ./test_output.txt
-tail -f ./test_output.txt &
-TAIL_PID=$!
-npx lerna run --ignore @cumulus/cumulus-integration-tests --concurrency 1 test:coverage > ./test_output.txt 2>&1
-RESULT=$?
-printf '\n\n\n*****TEST FAILURES:\n'
-grep '✖' ./test_output.txt
```

Comment:
this was removed because it's referenced only by the old test:ci script which isn't called anywhere by anything? this seems to be vestigial code. if that's not true let me know

Reply:
looking at bringing it back in and modifying it to wrap the upload to s3 and such